tests/Toolkit/VTest.php

```diff
@@ -458,6 +458,18 @@ public function testIp()
         $this->assertFalse(V::ip('192.168.255.24.23'));
     }
 
+    public function testJson()
+    {
+        $this->assertTrue(V::json('{"foo": "bar"}'));
+        $this->assertTrue(V::json('{}'));
+        $this->assertTrue(V::json('[]'));
+        $this->assertFalse(V::json('{foo: bar}'));
+        $this->assertFalse(V::json('foo'));
+        $this->assertFalse(V::json([]));
+        $this->assertFalse(V::json(false));
+        $this->assertFalse(V::json(42));
```

Comment:
json validator would be better in separate PR. But if it will add to this PR, a small unit test can be added for empty string:

```suggestion
        $this->assertFalse(V::json(42));
        $this->assertFalse(V::json(''));
```